Source/AliveLibAO/Abe.cpp

```diff
@@ -7885,9 +7887,322 @@ void Abe::State_87_428FA0()
     State_3_Fall_42E7F0();
 }
 
+ALIVE_VAR(1, 0x507728, int, gConter_507728, 0);
+ALIVE_VAR(1, 0x50772C, BYTE**, gpDemoPlaybackRes_50772C, 0);
+ALIVE_VAR(1, 0x507730, int, sAbeSound_507730, 0);
+ALIVE_VAR(1, 0x507724, BellSong*, sBellSong_507724, nullptr);
+
 void Abe::State_88_HandstoneBegin_430590()
 {
-    NOT_IMPLEMENTED();
+    switch (field_110_state)
+    {
+        case StoneStates::eUnknown_0:
+        {
+            if (field_10_anim.field_4_flags.Get(AnimFlags::eBit12_ForwardLoopCompleted))
+            {
+                ResourceManager::GetLoadedResource_4554F0(ResourceManager::Resource_Animation, 316, 1, 0);
+                field_164_pCircularFade = Make_Circular_Fade_447640(
+                    field_A8_xpos,
+                    field_AC_ypos,
+                    field_BC_sprite_scale,
+                    1,
+                    0
+                );
+
+                if (field_10_anim.field_4_flags.Get(AnimFlags::eBit5_FlipX))
+                {
+                    field_164_pCircularFade->field_10_anim.field_4_flags.Set(AnimFlags::eBit5_FlipX);
```

Comment:
you could just pass the .Get() into the 2nd arg of set instead, doesn't matter though